app/controllers/admin/comments/mark_as_spam_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::Comments::MarkAsSpamController < Admin::BaseController
```

Comment:
This bit us on another project recently, basically Rails' can't code load properly without explicit nesting because it might not have any idea what `Comments` is, a module or class. More here https://edgeguides.rubyonrails.org/autoloading_and_reloading_constants.html#nesting

Reply:
Oh, sorry, just saw this was `Outdated` so I'll dismiss.